client/js/screens/JoinUs/JoinUs.js

```diff
@@ -174,6 +185,7 @@ class JoinUs extends Component {
   }
 }
 
-export default compose(graphql(AUTHENTICATE_USER, { name: "signupMutation" }))(
-  JoinUs
-);
+export default compose(
+  graphql(AUTHENTICATE_USER, { name: "loginMutation" }),
+  withNavigation
```

Comment:
please pass navigation as a prop from the parent component instead of using withNavigation.